autopush_rs/src/client.rs

```diff
@@ -316,17 +327,30 @@ where
                 debug!("Got checkstorage response");
                 let webpush = self.data.webpush.as_mut().unwrap();
                 webpush.flags.include_topic = include_topic;
+                debug!("Setting unacked stored highest to {:?}", timestamp);
                 webpush.unacked_stored_highest = timestamp;
                 if messages.len() > 0 {
-                    webpush.flags.increment_storage = !include_topic;
-                    webpush.unacked_stored_notifs.extend(
-                        messages.iter().cloned(),
-                    );
-                    let message = ServerMessage::Notification(messages.pop().unwrap());
-                    ClientState::FinishSend(
-                        Some(message),
-                        Some(Box::new(ClientState::SendMessages(Some(messages)))),
-                    )
+                    // Filter out TTL expired messages
+                    let now = time::get_time().sec as u32;
```

Comment:
In rust, we use timestamps of <u32>, but python has timestamps of <u64>. Do we need to cast up to prevent possible overruns?